src/app/components/Image/index.tsx

```diff
@@ -79,22 +78,28 @@ const Image = ({
     aspectRatioY as number,
   );
 
-  const hasFallback = srcSet && fallbackSrcSet;
+  const hasFallback =
+    srcSet &&
+    fallbackSrcSet &&
+    (pageType === FRONT_PAGE || pageType === HOME_PAGE);
   const ImageWrapper = hasFallback ? 'picture' : Fragment;
   const ampImgLayout = hasDimensions ? 'responsive' : 'fill';
-
   const getImgSrcSet = () => {
     if (!hasFallback) return srcSet;
-    if (pageType !== FRONT_PAGE) return fallbackSrcSet;
+    if (pageType !== FRONT_PAGE && pageType !== HOME_PAGE)
+      return fallbackSrcSet;
     return undefined;
```

Comment:
```suggestion
    if (pageType !== FRONT_PAGE && pageType !== HOME_PAGE) {
      return fallbackSrcSet;
    }
    return undefined;
```
https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces